src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Game from './components/Game';
+
+const App = () => {
```

Comment:
at this moment we assume that App will return only the Game component. So, we can make this:
```
const App => () => <Game />
```